java/ql/src/semmle/code/java/frameworks/spring/SpringHttp.qll

```diff
@@ -38,3 +39,17 @@ class SpringResponseEntityBodyBuilder extends Interface {
 class SpringHttpHeaders extends Class {
   SpringHttpHeaders() { this.hasQualifiedName("org.springframework.http", "HttpHeaders") }
 }
+
+/** Models `org.springframework.http.RequestEntity`s instantiation expressions. */
+class SpringRequestEntityInstanceExpr extends ClassInstanceExpr {
+  int numArgs;
```

Comment:
Remove this field - this is already available as `. getNumArgument()`.

Reply:
Actually, strike that. Remove this entire class (i.e. inline `SpringRequestEntityInstanceExpr .getUriArg`).